src/airdrops/2-UNICORN.json

```diff
@@ -0,0 +1,42 @@
+{
+  "info": {
+    "name": "Unicorn",
+    "symbol": "UNICORN",
+    "contractAddress": "",
+    "network": "Cosmos",
+    "description": "ancient Windows 98 wizards, and neckbeard-fedora-wearing warriors",
+    "image": "https://unicorn.meme/img/uni-head.webp"
+  },
+  "claim": {
+    "dates": {
+      "claimDateOpen": "2024-04-02T00:00:00.000Z",
+      "claimDateClose": null,
+      "claimDuration": "unknown"
+    },
+    "value": {
+      "airdroppedTokens": null,
+      "airdroppedTokenDecimals": null,
+      "usdValueClaimOpen": null
+    },
+    "eligibility": {
+      "isAllocationScaledPerNft": null,
+      "derivsIncluded": [
+        "Yayo",
+        "Radbro",
+        "Milady",
+        "RedactedRemilioBabies",
+        "PixeladyMaker",
```

Comment:
formatte this, without the spaces, cause that was the vibe as far as i could tell? idk some of them look weird

Reply:
Nice, I reordered them in a more hierarchial order in my head and updated template.